src/test/java/com/coveo/FMTTest.java

```diff
@@ -47,6 +47,22 @@ public void withAllTypesOfSources() throws Exception {
     assertThat(fmt.getFilesProcessed()).hasSize(3);
   }
 
+  @Test
+  public void withAllTypesOfSourcesWithAospStyleSpecified() throws Exception {
+    FMT fmt = (FMT) mojoRule.lookupConfiguredMojo(loadPom("simple_aosp"), FORMAT);
+    fmt.execute();
+
+    assertThat(fmt.getFilesProcessed()).hasSize(3);
```

Comment:
This doesn't validate that the specified style was indeed used... I think it'd be doable to add a check for this by looking at the output, right?

Reply:
Thanks for the quick feedback! I enhanced the style tests to look at a spacing in the formatted files. I put assertions to make sure Google uses 2 spaces and AOSP uses 4.

Reply:
See my latest comment, and please don't hate me :grin:

Reply:
My point is that if the formatter uses 4 spaces (whereas you'd expect it to use only 2), the test that checks if the lines start with 2 spaces would still pass. It might be better to check that the resulting code does *not* contain a sequence of 4 spaces, for example...

Reply:
You're totally right... I wasn't thinking about the case where AOSP ran when Google was expected. I updated to check for the leading spaces and first key word. Hopefully this gets it!